travis/run-large-python-tests.sh

```diff
@@ -34,11 +34,12 @@ pytest --verbose tests/sagemaker/mock --large
 pytest --verbose tests/sklearn --large
 pytest --verbose tests/spark --large
 pytest --verbose tests/tensorflow/test_tensorflow_model_export.py --large
+pytest --verbose tests/tensorflow_autolog/test_tensorflow_autolog.py --large
 pip install 'tensorflow>=2.0.0rc2'
 pytest --verbose tests/tensorflow/test_tensorflow2_model_export.py --large
-pip install 'tensorflow==1.12'
+pytest --verbose tests/tensorflow_autolog/test_tensorflow2_autolog.py --large
+pip install 'tensorflow==1.14'
```

Comment:
I wonder if we could simplify things a bit & skip this step of re-installing TF 1.14 by moving running the tests below this line (`pytest  tests/azureml`, `pytest  tests/models`, etc) to before we install TF 2.0.0

Reply:
Just a heads up, will have to move the keras_autolog test up too, since they don't work with TF 2.0 as of now.

Reply:
Gotcha, will do